.cargo/config.toml

```
@@ -1,5 +1,5 @@
 [build]
 target-dir = "out/rust"
 [env]
-BORING_BSSL_PATH = { value = "vendor/boringssl-fips/linux_x86_64", force = true, relative = true }
-BORING_BSSL_INCLUDE_PATH = { value = "vendor/boringssl-fips/include/", force = true, relative = true }
+BORING_BSSL_FIPS_PATH = { value = "vendor/boringssl-fips/linux_x86_64", force = true, relative = true }
```

```suggestion
BORING_BSSL_FIPS_PATH = { value = "vendor/boringssl-fips/linux_x86_64", force = true, relative = true }
```
Does this also fix: https://github.com/istio/ztunnel/issues/399 ?


I haven't checked, but one of the minor annoyances before was we couldn't conditionally set this variable in `build.rs` per-arch/platform, because `boring-sys` was doing stuff in its own `build.rs` that both 

A) didn't work with vendoring
B) defeated cargo's "recompute" logic making it impossible to override in our `build.rs`

which means the build has to `sed` this and those of us on `linux_arm64` have to locally munge this file constantly.

---

I don't think so unless you buid with `ring` instead of boring which sidesteps the issue

---

Oh I forgot a bit part of this - I am proposing we default Istio builds to ring as well.. that decision can be decoupled from the broader PR though